homeworks/05-css-advanced/styles.css

```diff
@@ -0,0 +1,451 @@
+* {
+    box-sizing: border-box;
+}
+
+:root {
+    --bg-primary: #E9EBEC;
+    --bg-secondary: #FFFFFF;
+    --bg-terciary: #D7D7D7;
+    --bg-details: #FBD144;
+    --text-primary: #3D3D3D;
+    --text-secondary: #575757;
+    --text-terciary: #19191B;
+}
+
+[data-theme="dark"] {
+    --bg-primary: #0C151D;
+    --bg-secondary: #171F26;
+    --bg-terciary: #0C151D;
+    --bg-details: #FFE071;
+    --text-primary: #F1F2F4;
+    --text-secondary: #A3ABB2;
+    --text-terciary: #A3ABB2;
+}
+
+body {
+    margin: 0;
+}
+
+h1,
+h2,
+h3,
+p,
+span,
+.section__link {
+    font-weight: 500;
+    font-family: "Poppins", sans-serif;
+}
+
+.wrapper {
+    background-color: var(--bg-primary);
+    transition: all 2s linear;
+}
+
+.theme-transition {
+    transition: all 2s linear;
+}
+
+.container {
+    min-height: 100vh;
+    max-width: 1300px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem 3rem;
+    gap: 2rem;
```

Comment:
In this case better not to set generic gap, but to adjust distance between sections pointly according to design